package.json

```diff
@@ -1,122 +1,144 @@
 {
   "name": "fll-launcher",
-  "version": "1.3.10",
-  "description": "A Launcher to the FIRST LEGO League scoring system",
+  "version": "2.0.0",
+  "author": "Roy Shmueli <2903801+2roy999@users.noreply.github.com>",
+  "description": "A Launcher to the FIRST LEGO League tournament managment system",
   "main": "./dist/electron/main.js",
-  "author": "FIRST",
-  "license": "MIT",
+  "license": "GPL-3.0",
   "engines": {
-    "node": ">=8.2.1"
+    "node": ">=10.11.0"
   },
   "scripts": {
+    "build": "node .electron-vue/build.js && electron-builder",
+    "build:dir": "node .electron-vue/build.js && electron-builder --dir",
+    "build:clean": "cross-env BUILD_TARGET=clean node .electron-vue/build.js",
+    "build:web": "cross-env BUILD_TARGET=web node .electron-vue/build.js",
     "get": "node ./dev-scripts/get.js",
-    "start": "node .electron-vue/dev-runner.js",
     "lint": "eslint --ext .js,.vue -f ./node_modules/eslint-friendly-formatter src",
     "lint:fix": "eslint --ext .js,.vue -f ./node_modules/eslint-friendly-formatter --fix src",
-    "build:win": "cross-env NODE_ENV=production node .electron-vue/build.js && electron-builder -w --dir",
-    "build:linux": "cross-env NODE_ENV=production node .electron-vue/build.js && electron-builder -l --dir",
-    "build:clean": "cross-env BUILD_TARGET=clean node .electron-vue/build.js",
-    "build:web": "cross-env BUILD_TARGET=web node .electron-vue/build.js",
-    "postinstall": "electron-builder install-app-deps",
     "pack": "npm run pack:main && npm run pack:renderer",
     "pack:main": "cross-env NODE_ENV=production webpack --progress --colors --config .electron-vue/webpack.main.config.js",
-    "pack:renderer": "cross-env NODE_ENV=production webpack --progress --colors --config .electron-vue/webpack.renderer.config.js"
+    "pack:renderer": "cross-env NODE_ENV=production webpack --progress --colors --config .electron-vue/webpack.renderer.config.js",
+    "postinstall": "electron-builder install-app-deps",
+    "start": "chmod -R u+x . && node .electron-vue/dev-runner.js"
```

Comment:
This is not supported in windows, so this needs to be in mac/linux specific command